vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt

```diff
@@ -64,8 +64,6 @@ class PullScreenshotTestRunListener(
         val remoteDir = device.fileManager.getScreenshotsDir(applicationId)
         val outputDir = outputDirectory.toFile()
 
-        logger.trace { "Pulling screenshots from $remoteDir" }
```

Comment:
Again, this is only logged once. Does it make that much of a difference?

Reply:
It's logged once per test, and below we already log the dir name after pulling is finished.